packages/pwa-kit-dev/bin/pwa-kit-dev.js

```
@@ -183,7 +182,10 @@ const main = async () => {
     const appSSRjs = fse.pathExistsSync(appSSRpath)
     const overrideSSRpath = p.join(
         process.cwd(),
-        projectPkg?.mobify?.overridesDir ?? '',
+        typeof projectPkg?.ccExtensibility?.overridesDir === 'string' &&
+            !projectPkg?.ccExtensibility?.overridesDir?.startsWith('/')
+            ? '/' + projectPkg?.ccExtensibility?.overridesDir
+            : projectPkg?.ccExtensibility?.overridesDir ?? '',
```

I guess this answers one of my previous questions.

Should we warn the user when the defined `overridesDir` is not valid, instead of silently ignoring a value intentionally defined by the user?

---

Indeed, see my comment there 
https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1162#discussion_r1186237303